src/libraries/Microsoft.PowerFx.Core/Public/NameCollisionException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.PowerFx.Core
+{
+    public class NameCollisionException : Exception
+    {
+        public readonly string CollidingDisplayName;
```

Comment:
>CollidingDisplayName [](http://example.com/codeflow?start=31&length=20)

nit - this should be a field, not property. 

CollidingDisplayName  {get;private set; }